app/src/androidTest/java/net/skyscanner/backpack/chip/BpkOutlineChipTest.kt

```diff
@@ -30,29 +30,31 @@ import org.junit.runner.RunWith
 class BpkOutlineChipTest : BpkSnapshotTest() {
   @Before
   fun setup() {
-    setBackground(R.color.bpkPrimary)
+    setBackground(R.color.bpkBackgroundAlternative)
```

Comment:
set a different background colour here as with some chips you couldn't see the difference between background and chip making it impossible to verify